### PR TITLE
Class is-patch with background-color blue

### PR DESCRIPTION
**Pull request by beeva-josedongil**

### CODE REVIEW

**Comment by kmohrf** [comment]
Thanks for your contribution, but I’m not sure if it’s good to use the same color for both PUT and PATCH operations. Any other color suggestion?

File: assets/docs.less [hunk 1]
```diff
@@ -297,8 +297,12 @@ label.dropdown-item:hover {
         background-color: var(--oc-orange-6);
     }
 
+    &.is-patch {
+      background-color: var(--oc-grape-6);
```
Comment (beeva-josedongil) [inline assets/docs.less]: New color grape! :)

File: assets/docs.less [hunk 2]
```diff
@@ -297,8 +297,12 @@ label.dropdown-item:hover {
...
+    }
+  
     &.is-put {
-        background-color: var(--oc-blue-6);
+      background-color: var(--oc-blue-6);
```
Comment (kmohrf) [inline assets/docs.less]: you’ve accidentally changed indentation here

File: assets/docs.less (hunk 1 above)
Comment (kmohrf) [inline assets/docs.less]: this should be intended another two spaces as well

**Pull request closed without merging**